From dcc0fa1a8cb29b66bc5e2910bda6721765b40f35 Mon Sep 17 00:00:00 2001
From: Rakesh Iyer <riyer@nvidia.com>
Date: Thu, 28 Feb 2013 16:12:00 -0800
Subject: [PATCH] Dalvik heap size for a 10" hdpi tablet.

This is needed for 19x12 dalmore.

Bug 1241656.

Change-Id: I7ac0dcde708064fd93b1678f00c0a909b78f5b0c
Reviewed-on: http://git-master/r/205599
Tested-by: Rakesh Iyer <riyer@nvidia.com>
GVS: Gerrit_Virtual_Submit
Reviewed-by: Dan Willemsen <dwillemsen@nvidia.com>
---
 build/tablet-10in-hdpi-2048-dalvik-heap.mk | 22 ++++++++++++++++++++++
 1 file changed, 22 insertions(+)
 create mode 100644 build/tablet-10in-hdpi-2048-dalvik-heap.mk

diff --git a/build/tablet-10in-hdpi-2048-dalvik-heap.mk b/build/tablet-10in-hdpi-2048-dalvik-heap.mk
new file mode 100644
index 0000000..7c4644c
--- /dev/null
+++ b/build/tablet-10in-hdpi-2048-dalvik-heap.mk
@@ -0,0 +1,22 @@
+#
+# Copyright (C) 2012-2013 The Android Open Source Project
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+
+# Provides overrides to configure the Dalvik heap for a standard tablet device.
+
+PRODUCT_PROPERTY_OVERRIDES += \
+    dalvik.vm.heapstartsize=12m \
+    dalvik.vm.heapgrowthlimit=128m \
+    dalvik.vm.heapsize=448m \
-- 
1.8.1.5

